testing/fatutf8/fatutf8_main.c

```diff
@@ -108,6 +109,7 @@ int main(int argc, FAR char *argv[])
 
   printf("\n");
 
+  strcpy(dir_path, path);
```

Comment:
I do not think that removal of a directory unconditionally is a good idea. What if directory existed already and was intentionally created by user? The removal in such case will be confusing. Can we better check if directory is already created and delete only if it was created by this code?

Reply:
It's impossible since https://github.com/apache/incubator-nuttx-apps/pull/1063/files#diff-c6c45b54fda3ab97655d3becbfe72f45338ed829e17a019b868a6999e62ceb6cR98-R103 report EEXIST.

Reply:
Ok